tests/lang_exprs/ttype_expr_ast_corruption.nim

```diff
@@ -0,0 +1,20 @@
+discard """
+  description: '''
+    Ensure that a type expression can be used in the type slot of an object
+    construction syntax, which is itself the argument to a `[]` procedure
+    call
+  '''
```

Comment:
I don't know if it's a standard practice for nimskull to go like this but a link to the original issue would be helpful

Reply:
hah, we've not had issues so we're used to writing descriptions. 😄